packages/nimble-components/src/select/index.ts

```diff
@@ -1231,8 +1231,8 @@ export class Select
         this.$emit('filter-input', eventDetail, { bubbles: true });
     }
 
-    private maxHeightChanged(): void {
-        this.updateListboxMaxHeightCssVariable();
+    private availableViewportHeightChanged(): void {
```

Comment:
looking at this again, I think the reason for this updatee the observable property -> listen for Changed -> call style.setProperty is a relic from when we did not own the template.

We should now just update the template to do the style binding.

```
<div class="listbox" style="--ni-private-listbox-available-viewport-height: ${x => x.availableViewportHeight}px;"
```